src/platform/sdl/sdl-events.c

```diff
@@ -446,11 +446,9 @@ static void _mSDLHandleKeypress(struct mCoreThread* context, struct mSDLPlayer*
 			context->frameCallback = _pauseAfterFrame;
 			mCoreThreadUnpause(context);
 			return;
-#ifdef BUILD_PANDORA
```

Comment:
Please leave this in for now.

Reply:
I've dropped this change, however without it, there's no way of exiting mGBA when started in full-screen.

Reply:
Alt-F4? I guess it depends on the OS. Hm.